src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/Dashboard.test.tsx

```diff
@@ -61,12 +61,8 @@ import {
   DashboardUsPremiumScanInProgressUnresolvedBreaches,
 } from "./DashboardPlusUsers.stories";
 
-const routerPushMock = jest.fn();
-
 jest.mock("next/navigation", () => ({
-  useRouter: () => ({
-    push: routerPushMock,
-  }),
+  useRouter: jest.fn,
```

Comment:
Possibly it's not used if this didn't cause problems, but I think you forgot something here :)

```suggestion
  useRouter: jest.fn(),
```